src/components/send-money/SendMoney.js

```diff
@@ -113,6 +113,17 @@ class SendMoney extends Component {
         this.setState(() => ({
             [name]: value
         }))
+
+        if (name === 'accountId') {
```

Comment:
let's not use state to track computed properties, it makes it harder to follow data flow

e.g. here you can just add a getter for `requestStatusSameAccount`:

```
get requestStatusSameAccount() {
    if (this.props.accountId !== this.state.accountId) {
        return null;
    }
    return {
                         success: false,
                         messageCode: 'account.available.errorSameAccount',
                     }
}
```

then later just use it as `this.requestStatusSameAccount || requestStatus`

Reply:
done